### ISSUE TITLE
Double softmax in the synthetic experiments...

### ISSUE THREAD

**Issue by ogutsevda** [bifold-pathomics/xMIL#7]
Hi! I've realized that in the Classifier class, the validation step applies softmax (or sigmoid) to the logits and by default softmax is set to True and sigmoid is set to False. However, in the train_model() function, another softmax is applied, hence, there is double application of softmax. I believe this is problematic.

**Comment by hense96** [comment]
Many thanks for finding this issue and helping us improve the code! We have now addressed it in: https://github.com/bifold-pathomics/xMIL/pull/8. Please let us know if you face any other issues or have further suggestions :)
